mlrun/serving/v2_serving.py

```diff
@@ -527,6 +546,7 @@ def push(self, start, request, resp=None, op=None, error=None, partition_key=Non
                         "metrics",
                     ]
                     data["values"] = self._batch
+                    data["effective_sample_count"] = len(request["inputs"])
```

Comment:
In this case, I think you need to iterate over `self._batch` and sum up the `effective_sample_count` for each invocation. I'm not sure it was working well even before your change. :)

Reply:
fixed + identified a bug in the pre-defined batch param - https://iguazio.atlassian.net/browse/ML-8988